app/src/containers/MenuMobile.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
+import MenuMobile from '../components/Shared/MenuMobile';
+import { setVisibleMenu } from '../actions/appearence';
+
+const mapStateToProps = (state) => ({
+  menuVisible: state.appearance.menuVisible
```

Comment:
The MenuMobile knows and needs to know if it's open or closed, so it's always stateful. Difference is is the state is internal (this.state) or external (this.props). Nobody outside of the MenuMobile uses that state. External components only care about opening or closing it. So the interface to the component (mapDispatchToProps) should have an "open" and a "close" mappings that external elements (hamburguer icon) can call. They don't care about the state, neither should they

@clementprdhomme thoughts ?